chains/evm/contracts/ccvs/components/BaseOnRamp.sol

```diff
@@ -1,16 +1,16 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.24;
 
-import {ICCVOnRampV1} from "../interfaces/ICCVOnRampV1.sol";
-import {IRMNRemote} from "../interfaces/IRMNRemote.sol";
-import {IRouter} from "../interfaces/IRouter.sol";
+import {ICCVRampV1} from "../../interfaces/ICCVRampV1.sol";
+import {IRMNRemote} from "../../interfaces/IRMNRemote.sol";
+import {IRouter} from "../../interfaces/IRouter.sol";
 import {ITypeAndVersion} from "@chainlink/contracts/src/v0.8/shared/interfaces/ITypeAndVersion.sol";
 
 import {IERC20} from "@openzeppelin/contracts@4.8.3/token/ERC20/IERC20.sol";
 import {SafeERC20} from "@openzeppelin/contracts@4.8.3/token/ERC20/utils/SafeERC20.sol";
 import {EnumerableSet} from "@openzeppelin/contracts@5.0.2/utils/structs/EnumerableSet.sol";
 
-abstract contract BaseOnRamp is ICCVOnRampV1, ITypeAndVersion {
+abstract contract BaseOnRamp is ICCVRampV1, ITypeAndVersion {
```

Comment:
Same treatment as the other onramp/offramp merging? BaseVerifier that implements ICrossChainVerifierV1? I think we're ditching the Ramp part of naming for CCVs post-merge.